csro/oiwfs-env-assembly/command-model.conf

```diff
@@ -232,13 +232,13 @@ At Completion:
 <li> state.cmd = READY
 <li> state.temperature = WARMING | WARM
 </ul>"""
- }
+    }
 
- {
-   name             = setGateState
-   description      = """Instructs the OIWFS enclosure to open or close the gate valve.
+    {
+        name             = setGateState
+        description      = """Instructs the OIWFS enclosure to open or close the gate valve.
 
-This will tell the enclosure to change the state of the gate valve at the bottom of the snout.  The valve can either be set to be opened or closed and the result is dependent on the current conditions inside and outside of the enclosure.  If conditions will pose a hazard to the enclosure conditions (such as the enclosure is running cold and the OIWFS is not connected to NFIRAOS, then the command will be rejected.  This command will also be rejected if the environmental conditions pose a hazard (such as the temperature differentials greater than 0.5 degrees Celsius may cause frost/condensation).  When evaluating the environmental conditions, the OIWFS envlosure will consider not only the current conditions of the enclosure and gate valve, but also the current conditions of the snout, NFIRAOS (when connected) and ambient environment (when not connected to NFIRAOS).  
+This will tell the enclosure to change the state of the gate valve at the bottom of the snout.  The valve can either be set to be opened or closed and the result is dependent on the current conditions inside and outside of the enclosure.  If conditions will pose a hazard to the enclosure conditions (such as the enclosure is running cold and the OIWFS is not connected to NFIRAOS, then the command will be rejected.  This command will also be rejected if the environmental conditions pose a hazard (such as the temperature differentials greater than 0.5 degrees Celsius may cause frost/condensation).  When evaluating the environmental conditions, the OIWFS envlosure will consider not only the current conditions of the enclosure and gate valve, but also the current conditions of the snout, NFIRAOS (when connected) and ambient environment (when not connected to NFIRAOS).
```

Comment:
what about splitting this long line? it must be a pain in an editor

Reply:
I tend to use editors that wrap the line when displayed (vscode in this case). Not to say that you make a bad suggestion, but at the moment I don't want to go down the road of enforcing a particular formatting style on these model files.

Reply:
programmers discuss 80 vs 120 characters, here we have a line of 875 characters. ;)